### ISSUE TITLE
How does that project relate to new upcoming AngularJS 2.0?

### ISSUE THREAD

**Issue by Sergey80** [greencatsoft/scalajs-angular#39]
Considering that AngularJS 2.0 is a serial killer
https://twitter.com/mgonto/status/525206705156939776
How does that fit with this project road-map. Will it support / relate to / with AngularJS 2.0 to be more like Dart + Angular 2.0 but with Scala?

**Comment by mysticfall** [comment]
Honestly speaking, I can't say I have a clear vision about it at this moment.

I started this project for my own need, and as such, the development has been mostly following what comes up as necessary during my developing other services using it.

Probably, those services will inevitably need to be migrated to Angular 2.0, or to something entirely different, like WebComponents/Polymer, and it will be the time when I'll be revisiting this issue more seriously. However, I don't have any definite schedule or plan for the migration just yet.

I'm sorry for being so vague about it. Hopefully, I will be able to spend enough time to try AngularJS 2.0 soon.

**Comment by mdedetrich** [comment]
Made a comment here https://github.com/greencatsoft/scalajs-angular/issues/17#issuecomment-112250107

tl;dr. I would advise not to worry about it for another. Angular JS is a huge shift that is still moving quite fast.
